nuxeo-user-group-management/nuxeo-user-profile.html

```diff
@@ -35,7 +35,7 @@
 
 <dom-module id="nuxeo-user-profile">
   <template>
-    <style include="iron-flex iron-flex-alignment iron-flex-factors">
+    <style include="iron-flex iron-flex-alignment iron-flex-factors nuxeo-styles">
```

Comment:
UI-Elements should not rely on nuxeo-styles.
The reason why is because nuxeo-styles are declared on WebUI and nuxeo-ui-elements should be able to work without Web UI.
Probably what we want is to define the style here, in the element.

Reply:
[Example](https://github.com/nuxeo/nuxeo-ui-elements/blob/754e0c2aeb2157ecb7f0f1111f94140ba1196321/nuxeo-user-group-management/nuxeo-user-management.html#L173-L184)